bin/compile

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
It looks like you're trying to convert this repository to a buildpack. But this repository is not in the chain of buildpack (see `.buildpacks` file).
Plus wouldn't it be better to backport those changes into the metabase buildpack and not create a new one ?

Reply:
Ok, I didn't think this would be an issue. But I understand your point, closing for now.